config/locales/en/images/edit.yml

```diff
@@ -3,6 +3,7 @@ en:
     edit:
       title: "Image details for ‘%{title}’"
       title_modal: "Image details"
+      alt_text_placeholder: Your uploaded image
```

Comment:
Do we want to make this more of a call to action? e.g. "You must provide an alt value for this image", or "Describe this image to screen readers".

Reply:
This content was provided by our content designer. Happy for it to be something else - I would recommend challenging this on the Trello card.

Reply:
As it's not in the form but on the image itself, this is fine. I misunderstood earlier.